### PR TITLE
Attemps to add ci in Radium-Apps + fix radium apps

### PR DESCRIPTION
**Pull request by nmellado**
Fix linking error on windows (see https://github.com/STORM-IRIT/Radium-Engine/pull/671)

### CODE REVIEW

**Comment by nmellado** [comment]
@dlyr have you seen this ??

https://github.com/STORM-IRIT/Radium-Apps/runs/1777934855?check_suite_focus=true#step:11:508

**Comment by dlyr** [comment]
That's funny :)

**Comment by nmellado** [comment]
My bet is that this error is caused by the cached version of the externals, like if the compiler has changed from the previous run.

**Comment by nmellado** [comment]
At least I can reproduce the error observed on Radium-Release: https://github.com/STORM-IRIT/Radium-Apps/runs/1777934821#step:14:79

**Comment by dlyr** [comment]
key: ${{ runner.os }}-QtCache 
The cache key need to have the compiler
something like 
   key: ${{ runner.os }}-${{ config.cc }}-QtCache

**Comment by dlyr** [comment]
It works on Radium-Engine since we don't have workflow with same runner and different cc

File: .github/workflows/pull-request-ci.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      - name: Clone Radium
+        run: |
+          cd src && git clone https://github.com/STORM-IRIT/Radium-Engine.git --single-branch && cd ..
+
+      - name: Cache externals
+        id: cache-external
+        uses: actions/cache@v2
+        with:
+          path: external
+          key: ${{ runner.os }}-${{ config.cc }}-external-${{ hashFiles('src/Radium-Engine/external/**/CMakeLists.txt') }}
```
Comment (dlyr) [inline .github/workflows/pull-request-ci.yml:64]:
```suggestion
          key: ${{ runner.os }}-${{ matrix.config.cc }}-external-${{ hashFiles('src/Radium-Engine/external/**/CMakeLists.txt') }}
```

File: .github/workflows/pull-request-ci.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      - name: Add msbuild to PATH
+        uses: seanmiddleditch/gha-setup-vsdevenv@master
+        if: runner.os == 'Windows'
+
+      - name: Cache Qt
+        id: cache-qt
+        uses: actions/cache@v2
+        with:
+          path: ../Qt
+          key: ${{ runner.os }}-${{ config.cc }}-QtCache
```
Comment (dlyr) [inline .github/workflows/pull-request-ci.yml:38]:
```suggestion
          key: ${{ runner.os }}-${{ matrix.config.cc }}-QtCache
```

**Review by dlyr — COMMENTED** [review]
oups

**Comment by nmellado** [comment]
Thanks

**Comment by nmellado** [comment]
Ready to review @dlyr

**Pull request merged**